Tour/basic-types.md

```diff
@@ -5,15 +5,15 @@ Hylo is statically typed: the type of a binding must always match the type of th
 ```
 public fun main() {
   var length = 1
-  &length = 2.3 // error: expected type `Int`, found `Double`
+  &length = 2.3 // error: type 'Int' does not conform to trait 'ExpressibleByFloatLiteral'
```

Comment:
```
.build/release/hc Examples/test.hylo -o test && ./test
Examples/test.hylo:3.13-16: error: type 'Int' does not conform to trait 'ExpressibleByFloatLiteral'
  &length = 2.3 // error: type 'Int' does not conform to trait 'ExpressibleByFloatLiteral'
```

Reply:
Hmm, inference did the wrong thing here. The original message is "more correct". Could you open an issue in the compiler repo about this?

Reply:
Created https://github.com/hylo-lang/hylo/issues/1291
Feel free to change the title if I've not captured the core of the issue well.

Reply:
Reverted error message, but with the update of `Double` -> `Float64`. https://github.com/hylo-lang/GitBooks/pull/13/commits/3558d9c100c214cc3037da0c515c19719c3b6a6b